README.md

```diff
@@ -52,6 +52,18 @@ There are 4 key features we plan to implement in the near term.
 
 For more detail on these features and where we are going, see [our roadmap](ROADMAP.md).
 
+## supported and experimental usage modes
+
+_cellxgene_ is built on standard web technologies, but is currently designed as a single-user desktop application. 
+Some of our users have experimented with hosting _cellxgene_ either for their lab or for public use, but please note that the _cellxgene_ team does not officially support, troubleshoot, or maintain any web deployments at this time.
+
+If you choose setup _cellxgene_ as a hosted service, you should be aware of the following risks:
+
+- `$ cellxgene launch` uses Flask's development server, which is not recommended for hosted deployment (see the [Flask documentation](http://flask.pocoo.org/docs/1.0/tutorial/deploy/#run-with-a-production-server))
+- We have no testing or official support for deployments where multiple users are accessing the same _cellxgene_ instance.
+- Your _cellxgene_ instance is likely to fall over if too many people access it at the same time, especially if they using functions that call the Python backend (such as differential expression, updating the layout, or coloring by gene).
```

Comment:
Can we find a more technical term than "fall over"? I can't think of the right word at the moment, but maybe it will come to me tomorrow.

Reply:
Is the issue that the server runs out of RAM because it's trying to repeatedly load data into memory to do the diffexp computation?

Reply:
hang?

Reply:
hang sounds good. It's the cpu, not the memory in diffexp case, I believe.